### ISSUE TITLE
Does not operate normally in build mode

### ISSUE THREAD

**Issue by seoyeon01** [alelievr/HDRP-UI-Camera-Stacking#18]
Unity : 2021.3.0f1

There is no problem in editor mode.
However, there is a problem with the runtime after the build. (exe)

Nothing is drawn on the screen.
The screen is black.

To be exact, the screen visible through the camera is not rendered.
The UI, which is ScreenSpace Overlay, is rendered.

Disabling the UI Camera Object at runtime will draw the screen normally.
When reactivated, the screen turns black.
There is no particular error message.

In order to accurately communicate the cause of this problem,
I tried to create a similar environment in a clean project, 
but it was not reproduced. (It works normally even at runtime)

Variables not included in the Clean Project are:
- Addressable
- Multi-Scene Structure
- Shader and Art Resources etc 
I didn't check including these, 
but I thought these were not the cause of the problem.
(Can these affect?)

Which part should I check?

**Comment by seoyeon01** [comment]
Adds information from the frame debugger.
![image](https://user-images.githubusercontent.com/48429544/189132149-5e5b3fa5-5539-4eca-8a62-20db46e8d332.png)
Problem occurs through the Composite UI Camera Stacking process.
Before Composite UI Camera Stacking process, all results seem to be the same and normal.
In Editor Mode, UI Camera and Main Camera images are well combined in Composite UI Camera Stacking.
but In Runtime Build mode,
The screen turns black during the composite UI Camera stacking process.

**Comment by seoyeon01** [comment]
Problem solved.
Added Hidden/HDRP/UI_Compositing to Always Included Shaders.
Is it okay to solve the problem like this?

**Comment by alelievr** [comment]
Yes it's fine to fix the issue this way. Though the shader should already be included in the build as I tried to serialize it in the HDCameraUI component 🤔

**Comment by seoyeon01** [comment]
I think the problem was caused by how to use the addressable.
If it's okay to use it this way,
I think it'd be good to do a close issue.
Thank you for your quick and thank you for making a good project! 
It was a great help.

+ ps
It may be a silly question,,,, I have two questions.
SpriteRenderer, not Canvas UI, seemed to be drawing well, too.
- Can I use SpriteRenderer with this(not Canvas UI)?
- Is it possible to apply it as a way to draw 3D objects?
It seemed to have the RenderTexture function
Is this possible?

**Comment by alelievr** [comment]
Hey, sorry for the delay, I didn't see the message.

> Can I use SpriteRenderer with this(not Canvas UI)?

As you can see here in the code: [HDCameraUI.cs:166](https://github.com/alelievr/HDRP-UI-Camera-Stacking/blob/master/Assets/com.alelievr.hdrp-ui-camera-stacking/Runtime/HDCameraUI.cs#L166) this package renders all the transparent objects of the scene if they match the specified layer. So yes you should be able to use it to render 3D transparent objects and sprites.

> Is it possible to apply it as a way to draw 3D objects?

Yeah, though with the current filtering options in the DrawRenderers() it will only pick transparent objects. Also, it's not possible to have lighting on your 3D objects if you render them through this package.
